backend/src/index.ts

```diff
@@ -19,7 +19,15 @@ app.get("/api", async (req, res) => {
 });
 
 app.get("/api/dogs/info", async (req, res) => {
-  res.json({ dogs: DOGS });
+  const client = await pool.connect();
```

Comment:
In `HomePage.tsx`, call this endpoint to fetch all the dogs' info.

Reply:
Done